anko/library/static/commons/src/ContextUtils.kt

```diff
@@ -65,21 +70,25 @@ inline val Activity.act: Activity
 inline val Activity.contentView: View?
     get() = findOptional<ViewGroup>(android.R.id.content)?.getChildAt(0)
 
-inline fun <reified T : View> View.find(@IdRes id: Int): T = findViewById(id) as T
-inline fun <reified T : View> Activity.find(@IdRes id: Int): T = findViewById(id) as T
-inline fun <reified T : View> Fragment.find(@IdRes id: Int): T = view?.findViewById(id) as T
-inline fun <reified T : View> Dialog.find(@IdRes id: Int): T = findViewById(id) as T
+inline fun <reified T : View> View.find(@IdRes id: Int): T = findViewById(id)
```

Comment:
That's incorrect because the return type is nonnull:
```
 LINENUMBER 162 L2
    ALOAD 1
    ILOAD 2
    INVOKEVIRTUAL android/view/View.findViewById (I)Landroid/view/View;
    DUP
    LDC "findViewById(id)"
    INVOKESTATIC kotlin/jvm/internal/Intrinsics.checkExpressionValueIsNotNull (Ljava/lang/Object;Ljava/lang/String;)V
```